### DISCUSSION TITLE
Tombstone Table Cell Spacing / Text Alignment

### DISCUSSION THREAD

**Discussion by chrisdaaz** [thegetty/quire#705] [Get Help] [answered]
I'm having trouble figuring out how to fix an alignment issue when there are multiple tombstones on a catalog entry page. It would be nice to make them consistent. Any ideas on where I can check to make adjustments?

<img width="832" alt="image" src="https://user-images.githubusercontent.com/24395592/230445754-dc0ba822-4102-425d-987f-529afa34bc42.png">

Live demo: http://life-and-the-afterlife.s3-website.us-east-2.amazonaws.com/catalog/daily-life-in-ancient-egypt/3/

Thanks!

**Comment by geealbers** [comment]
@chrisdaaz, yes, I see what you mean. The width of the table columns are being automatically determined by the browser, which tries to accommodate the content of every cell as best as it can. So different content in different tombstones leads to different column widths.

You can make them consistent across all tombstone tables with a little bit of CSS, which can be added to the project's `content/_assets/styles/custom.css` file:

```css
.quire-entry__tombstone table td:first-child {
  width: 6rem;
}
```

This will set the width of the first cell in every row. I've used `rem` units here so it's relative to the width of the type, but you could also use other units. And note that this will affect the entire book, not just this page.

**Reply by chrisdaaz** [reply, marked as answer]
Thank you! This worked for me!

```css
.quire-entry__tombstone table tr td:first-child {
    width: 6rem;
}
```
